InnoClinic.AuthorizationAPI/BLL/Models/DTOs/RegisterDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿using DAL.Constants;
+
+namespace BLL.Models.DTOs;
+
+public sealed record class RegisterDTO 
+{
+    public string Email { get; set; }
+    public string Password { get; set; }
+    public string ReEnteredPassword { get; set; }
+    public string Role { get; set; } = Roles.Patient;
```

Comment:
Why `Role` is a `string` type?

Reply:
My opinion:
Using strings for roles gives us:
Direct compatibility with Identity.
Ease of working with API/database without conversions(If we use enum, we will have to convert it to a string, which adds unnecessary code).

Reply:
```suggestion
    public string Role { get; set; } = Roles.Patient;
```
By the way, here you already make a conversion

Reply:
sorry dude, but we'll stick to another approach